agent/metasploit_agent.py

```diff
@@ -84,16 +84,23 @@ def process(self, message: m.Message) -> None:
             logger.info("Selected metasploit module: %s", selected_module.modulename)
             targets = utils.prepare_targets(message)
             for target in targets:
-                rhost = target.host
+                vhost = target.host
```

Comment:
Some metasploit modules will need and RHOST (IP) along with a VHOST (Hostname) to work properly, that's why I was setting them separately.